### DISCUSSION TITLE
Accessing metadata

### DISCUSSION THREAD

**Discussion by Yattabyte** [apache/superset#16584] [Q&A / Help]
Is it possible at all to access the metadata for where charts are stored, either to programatically create or modify charts?

I've been using superset for about a year now, and have at least 26 pages of charts that we actively use. If I want to do some database maintenance and rename a bunch of columns for example, either I or someone else would have to manually correct every affected chart.

**Comment by kamalkeshavani-aiinside** [comment]
Check `SQLALCHEMY_DATABASE_URI` in your config file. This is the db where all the Superset metadata is stored.
You will have to edit the values in this db to achieve what you want to do.(maybe not the best way)
I would suggest to take a backup of metadata db, before you manually edit it.

I am not sure, but I think you can also use swagger api to modify your charts. Check here
https://superset.apache.org/docs/rest-api
